llpc/test/shaderdb/gfx10/PipelineGs_PlaceMergedShaderFirst.pipe

```diff
@@ -0,0 +1,36 @@
+; Test that the first shader in the text section is the "GS" or "ES" shader.  This is needed when linking unlinked shaders.
```

Comment:
Is there a better way to get a test to run with GFX9 and GFX10 only?  I have two copies of the same test, which does not seem reasonable.

Reply:
You could have a RUN line with `-gfxip=9` and one with `%gfxip` (in the gfx10 folder). It is a bit hacky though.